internal/controller/capacity.go

```diff
@@ -191,10 +216,68 @@ func (*InstasliceReconciler) getStartIndexFromPreparedState(instaslice *inferenc
 		}
 
 	}
-
 	return newStart
 }
 
+func (r *InstasliceReconciler) actualProfileSliceSize(ctx context.Context, instaslice *inferencev1alpha1.Instaslice, profileName, gpuUUID string) (int32, error) {
```

Comment:
Can we consolidate these two methods `actualProfileSliceSize`,  `getTotalFitForProfileOnGPU` ?